### DISCUSSION TITLE
Inquiry About Inline Cell Editing in Simple DataTables

### DISCUSSION THREAD

**Discussion by cl3mcg** [fiduswriter/Simple-DataTables#411] [Q&A]
Hello,

I am trying to achieve the "Editable" table functionality as demonstrated in the example on the [demo page](https://fiduswriter.github.io/simple-datatables/demos/15-editing/). However, I couldn't find any specific documentation or section that explains how to implement inline cell editing.

Is there a particular option or value that needs to be passed to enable this feature?

Upon inspecting the DOM of the demo page, I noticed the use of a `makeEditable` function and specific attributes. Unfortunately, there is no mention of these in the documentation, or I might have missed it.
It seems that it would be related to some of the events that can be listened to such as [`editable.init`](https://fiduswriter.github.io/simple-datatables/documentation/Events#editableinit) or [`editable.save.cell`](https://fiduswriter.github.io/simple-datatables/documentation/Events#editablesavecell) that are documented, but there is no mention of the editor in itself.

Could you please guide me on how to enable inline cell editing or point me to the relevant documentation?

Thank you.
Best regards,

**Comment by johanneswilm** [comment]
Hey @cl3mcg,
the idea with the demo pages is that you use the "show source" functionality of your browser to see how it is done. Most developers prefer this way of documenting the usage. 

It sounds like you instead inspected the DOM. That may be less useful as you will see dom changes caused by this library that you may not make sense to use. 

The "editable table" is a plugin that is not part of the core functionality, which is why you don't see it in the documentation.

**Comment by johanneswilm** [comment]
@cl3mcg If you want to write a documentation page for the `makeEditable` function, I'd accept a PR for that.
